Makefile

```diff
@@ -4,6 +4,7 @@ FULL_VERSION=$(VERSION)+b$(CIRCLE_BUILD_NUM)
 BASE_VERSION=$(shell python ./manage.py version | cut -d + -f 1)
 # VERSION gets evaluated every time it's referenced, therefore we need to use VERSION here instead of FULL_VERSION.
 FILENAME=$(CIRCLE_ARTIFACTS)/$(NAME).$(VERSION).tar.gz
+TEST_FILE?=tests/
```

Comment:
It can be used to pass any argument/option to `nosetsts`; maybe we should give it a more generic name, like `TEST_ARGUMENTS` or something like this?

Reply:
Thanks for the feedback. I think it's a good point, we can rename it as `TEST_ARGS` or `TEST_ARGUMENTS` but the variable has to be quoted like;
`docker-compose run --rm -e TEST_ARGS="--process=4 tests/handlers/test_dashboards.py" server tests`

Passing this argument will override the entire default arguments. The default one will be `--with-coverage --cover-package=redash tests/`

Does this make sense?